extensions/blocks/anchor-fm/anchor-fm.php

```diff
@@ -110,8 +118,13 @@ function process_anchor_params() {
 		$data['spotifyShowUrl'] = $spotify_show_url;
```

Comment:
Should this remain here, now that we have it within the `insert-spotify-badge` action as well?

```suggestion
```

Reply:
I decided to move these variables to the `actions` array. In an action depends on a variable can be defined as an array where the first item is the name of the action, and the second item is an object with all the variables that the action depends on:

https://github.com/Automattic/jetpack/blob/37926847d08147460ff87640387931bf9d97fbac/extensions/blocks/anchor-fm/editor.js#L85

Since those query params are not used anywhere else, I removed them from global data object. If we identify scenarios where this data should be available out of the actions, we can add them back (or even if there are multiple actions that share the same data).

It's something that can be iterated according to our needs.